docs/src/changelog.adoc

```diff
@@ -2,6 +2,28 @@
 
 This document shows the changes that are introduced into Smarti with the releases.
 
+=== 0.6.0
+
+==== Upgrading
+
+With `0.6.0`, the datamodel has changed so data from older installations
+must be migrated to the new model.
+
+A migration script is available and must be applied manually:
+[source,javascript]
```

Comment:
Maybe include or link `dist/src/main/resources/migration/0.5-to-0.6.js` instead of duplicating it in the Documentation

Reply:
@ja-fra saw you added that. 👍